data/json/items/gun/545x39.json

```diff
@@ -9,19 +9,19 @@
     "price": 390000, "//" : "AKs likewise aren't commercially traded in the US, plus this is newer.",
     "bashing": 11,
     "to_hit": -1,
-    "material": ["steel", "wood"],
+    "material": [ "steel", "wood" ],
```

Comment:
Never mix unrelated JSON cleanup with behavioral changes or other migrations. This just increases line count.